static/js/twitter-filter.js

```diff
@@ -0,0 +1,30 @@
+function twitter_filter() {
+  var twitter_handles = []
+  for (var i = 0; i < document.getElementsByClassName('org-info').length; i++) {
+    var org_info = document.getElementsByClassName('org-info')[i]
+    for (var j = 0; j < org_info.children.length; j++) {
+      if (org_info.children[j].href !== undefined) {
+        if (org_info.children[j].href.startsWith('https://twitter.com/')) {
+          twitter_handles.push(org_info.children[j].href.substr(20))

```

Comment:
That `20` number is not clear. Can you create a constant or a variable like `twitter_prefix`? Add `twitter_username` and assign to that `blabla.href.substr(twitter_prefix)` would be more readable.